gumroad/views/admin.php

```diff
@@ -125,17 +143,17 @@
 	<h4>More examples</h4>
 
 	<ul class="ul-disc">
-		<li><code>[gumroad id="DviQY" button="true" type="overlay" text="Purchase Item" wanted="true"]</code></li>
+		<li><code>[gumroad url="https://gumroad.com/l/DviQY" button="true" type="overlay" text="Purchase Item" wanted="true"]</code></li>
 		<p><img src="https://s3.amazonaws.com/gumroad/assets/wordpress_docs/wantedoverlaydemo.gif"></p>
 	</ul>
 
 	<h3>Automatic linking and embeds</h3>
 
-	<p>You can forego using shortcodes entirely, and links to Gumroad products will automatically trigger the overlay. Embed code copied from the Gumroad product widget will also work without including the script tag:</p>
+	<p>You can forego using shortcodes entirely, and links to Gumroad products will automatically trigger the overlay. Embed code copied from the <a href="https://gumroad.com/widgets" target="_blank">Gumroad product widget</a> will also work without including the script tag:</p>
 
 	<p>
 		<code>
-		&lt;div class="gumroad-product-embed" data-gumroad-product-id="DviQY"&gt;&lt;a href="https://gumroad.com/l/qLlJJ"&gt;Loading...&lt;/a&gt;&lt;/div&gt;
+		&lt;div class="gumroad-product-embed"&gt;&lt;a href="https://gumroad.com/l/DviQY"&gt;Loading...&lt;/a&gt;&lt;/div&gt;
```

Comment:
A screenshot of the preview of this page is posted in the description of the PR.